routes/api/comments.js

```diff
@@ -34,7 +34,10 @@ router.post('/comments/:id/likes', restoreUser, asyncHandler(async(req, res) =>
 
 
 //Unlike a comment
-router.delete('/comments/likes/:id', asyncHandler(async(req, res) => {
+router.delete('/comments/likes/:id', restoreUser, asyncHandler(async(req, res) => {
+    const likedComments = await CommentLike.findAll();
+    const like = likedComments.some(likedComment => likedComment.id === parseInt(req.params.id));
```

Comment:
This can be simplified to
```js
const like = CommentLike.findByPk(req.params.id);
```

Reply:
Lines 38 and 39 simplified to "const like = CommentLike.findByPk(req.params.id);" on Line 29.